src/framework/vst/internal/vstplugin.cpp

```diff
@@ -112,6 +112,17 @@ PluginView VstPlugin::view()
     return m_pluginView;
 }
 
+PluginComponentPtr VstPlugin::component()
+{
+    if (m_pluginComponent) {
+        return m_pluginComponent;
+    }
+
+    m_pluginComponent = m_pluginProvider->getComponent();
+
+    return m_pluginComponent;
+}

```

Comment:
I think, better method make `const` and for m_pluginComponent add `mutable`

Reply:
yep, also makes sense for view() method